### ISSUE TITLE
Running under cron

### ISSUE THREAD

**Issue by ehud** [kootenpv/yagmail#6]
using yagmail from the shell is lovely! But when I try to run the script through cron it seems to fail trying to connect to the keyring. The only approach I found seems quite complicated (I haven't tried it yet):

http://jason.the-graham.com/2011/01/16/gnome_keyring_with_msmtp_imapfilter_offlineimap/#cron

**Comment by kootenpv** [comment]
I'm not able to reply somehow.

**Comment by kootenpv** [comment]
Hey, indeed I also went down that path unsuccesfully. In the end I just made an extra emailadress for which I didn't mind if the password would leak.

I gave it another look yesterday (and wrote a long post, but GitHub somehow prevented the reply).

The main thing was that I was again unsuccessful. The only lead I have is that perhaps this approach might work (also have not been able to test it yet, not on ubuntu, but it sounds very promising):

http://stackoverflow.com/a/29507718/1575066

I do very much appreciate though that you dropped a line!

Best,
Pascal
